### ISSUE TITLE
Specify which database backend to use

### ISSUE THREAD

**Issue by RealOrangeOne** [RealOrangeOne/django-tasks#38]
It's currently not possible to specify which database backend will be used (ie the ones configured in `DATABASES`). Especially for SQLite, it's useful to separate the queue tables into a separate database.

**Comment by turingnixstyx** [comment]
Hey is this still open and unassigned?

**Comment by RealOrangeOne** [comment]
Yes. If you're interested in working on it, by all means do. I don't assign issues to people specifically though - anyone can work on anything.

**Comment by fazeelghafoor** [comment]
@RealOrangeOne if this needs to be implemented, can we just add ```database``` in the following tasks config if we are using ```DatabaseBackend``` and set the default to default database defined in settings and then have all task related models to use the database defined in tasks config dict.
```python
TASKS = {
    "default": {
        "BACKEND": "django_tasks.backends.immediate.ImmediateBackend",
        "DATABASE": "name_of_the_database" i.e. "tasks_db" # if not set use the default
    }
}
```
or are we looking for a more sophisticated solution than this option.

**Comment by RealOrangeOne** [comment]
I think it'll need something a little more complex than this, as both the worker and migrations need to use this value. It might be that a [database router](https://docs.djangoproject.com/en/5.0/topics/db/multi-db/#automatic-database-routing) is the only way to implement this, in which case there's not much to do ourselves besides ensure the correct DB is specified when needed.

**Comment by RealOrangeOne** [comment]
I think a better way to handle this would be using a database router. I don't think one should be included, but at least documenting how to write one would be ideal. Exactly how multiple installed database backends would be handled I don't know.

**Comment by grandimam** [comment]
Hi @RealOrangeOne , I have created a draft PR that attemps to solve the problem based on the above suggestions. Any suggestions would be helpful - https://github.com/RealOrangeOne/django-tasks/pull/122.
